### DISCUSSION TITLE
Sitemap: Is link tag not needed?

### DISCUSSION THREAD

**Discussion by fabianwohlfart** [harlan-zw/nuxt-seo#321] [Q&A]
Hey,

I am missing <link rel="sitemap" type="application/xml" title="Sitemap" href="/sitemap_index.xml"> in my generated html. Is it not necessary, or must I add it myself? Cannot find anything about it in the docs.
Appreciate the help!

**Comment by harlan-zw** [comment]
Hi, sorry what do you mean a link tag? The sitemap is linked from the robots.txt or you need to manually add it to google search console.
